assets/src/blocks/Timeline/TimelineFrontend.js

```diff
@@ -0,0 +1,29 @@
+import { Fragment } from '@wordpress/element';
+import { TimelineNode } from './TimelineNode';
+
+export const TimelineFrontend = (props) => {
+	const {
+		timeline_title,
+		description,
+		isSelected,
+		...nodeProps
+	} = props;
+
+	return (
+		<section className="block timeline-block">
+			{
+				timeline_title
+					? <header>
```

Comment:
We could also write this as ```{ timeline_title && <header>...</header> }```

Reply:
That way it's consistent with this line https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/368/files?diff=split&w=1#diff-70066a6561381bd819c059627c0e34faR117